content/docs/capabilities/self-hosted-authenticate-service.md

```diff
@@ -8,66 +8,82 @@ description: Use Pomerium's Self-Hosted Authenticate Service to set up and deplo
 
 # Self-Hosted Authenticate Service
 
-Pomerium's **Self-Hosted Authenticate Service** allows you to configure your own **Authenticate Service URL** and **Identity Provider**.
+Pomerium's **Self-Hosted Authenticate Service** allows you to configure your own authenticate service URL and identity provider.
 
-The Self-Hosted Authenticate Service is available for both open-source Pomerium Core and Enterprise users.
+The self-hosted authenticate service is available in Pomerium Zero, open-source Pomerium Core, and Pomerium Enterprise.
 
-## How the Self-Hosted Authenticate Service works
+## How the self-hosted authenticate service works
 
-Self-hosting your Pomerium instance requires you to configure an [Authenticate Service URL](/docs/reference/service-urls#authenticate-service-url) and an OIDC-compliant [identity provider](/docs/identity-providers) (IdP).
+Self-hosting your Pomerium instance requires you to configure an [authenticate service URL](/docs/reference/service-urls#authenticate-service-url) and an OIDC-compliant [identity provider](/docs/identity-providers) (IdP).
```

Comment:
Side note: while most supported identity providers are in fact OIDC-compliant, a couple are not (GitHub and Apple).